shell/edit/workload/index.vue

```diff
@@ -145,23 +152,37 @@ export default {
 
     const spec = this.value.spec;
     let container;
-    const podTemplateSpec = type === WORKLOAD_TYPES.CRON_JOB ? spec.jobTemplate.spec.template.spec : spec?.template?.spec;
+    const podTemplateSpec =
+      type === WORKLOAD_TYPES.CRON_JOB ? spec.jobTemplate.spec.template.spec : spec?.template?.spec;
     let containers = podTemplateSpec.containers;
 
-    if (this.mode === _CREATE || this.mode === _VIEW || (!createSidecar && !this.value.hasSidecars)) {
+    if (
+      this.mode === _CREATE ||
+      this.mode === _VIEW ||
+      (!createSidecar && !this.value.hasSidecars)
```

Comment:
Formatting changes from ESLint

Reply:
@catherineluse I think you may have some custom preferences / extensions that are making these changes. We don't have any `max-len` rule and I don't think there's a default. Other's aren't hitting this either and these lines are now too short.